code/modules/spacepods/spacepod.dm

```diff
@@ -148,6 +148,12 @@
 	cargo_hold.max_w_class = 5		//fit almost anything
 	cargo_hold.max_combined_w_class = 0 //you can optimize your stash with larger items
 
+	for(var/turf/T in locs)
```

Comment:
I am fully aware this is a shitty way to do it, but it's the only way I could delete the spacepod frame after it was created